UIPS/UIP-0131.md

```diff
@@ -0,0 +1,121 @@
+
+The protocol implements rate limiting, mandatory cooldowns, and minimum confirmation periods to prevent timing attacks. A monitoring client watches both chains for fraudulent tickets and automates violation reporting.
+
+TODO: Specify signature thresholds, timeouts, rate limits, and violation report format.
+
+### Arvo Integration
+
+The introduction of multi-chain state requires significant changes to Urbit's PKI verification system. Currently, %jael maintains Azimuth state by synthesizing data from Ethereum and the naive rollup. The addition of Base requires extending this system to handle three distinct sources of truth.
+
+The %base-watcher agent extends %jael's PKI verification capabilities, monitoring Base chain activity and updating Azimuth state accordingly. While the Ministry bridge protocol handles transfer sequencing and validation through its vault mechanism and oracle system, %jael remains responsible for maintaining a consistent view of the PKI state. This includes tracking point locations across chains and handling state rollbacks that may occur before bridge transfers are finalized.
+
+In cases of rollbacks, %jael must ensure that its state synthesis algorithm appropriately handles the reversion without compromising the integrity of cross-chain point locations. This becomes particularly important when points are in transit between chains.
+
+TODO: Detail exact state synthesis algorithm for handling multi-chain state reconciliation.
+
+### Base Implementation
+
+Base's optimistic rollup architecture introduces specific timing requirements for the return path of the bridge protocol. While points can begin operating immediately upon arrival on Base, returns to Ethereum require waiting for Base's 7-day challenge period to expire. This ensures the Embassy contract state is final before the Ministry vault releases points back to Ethereum.
+
+Oracle validation windows adapt to Base's block confirmation schedule, requiring a minimum of 15 L2 blocks before accepting transfer tickets. The Embassy contract implements a local timeout mechanism to handle potential state synchronization issues between Base and Ethereum during return transfers.
+
+TODO: Specify exact block confirmation thresholds and timeout parameters.
+
+Rationale
+---------
+
+The design establishes Ethereum as the senior chain to preserve the existing security model while enabling point mobility. Using galaxies as oracles leverages the network's existing power structure and economic incentives rather than introducing external validators. Points maintain complete functionality across chains through comprehensive state transfer.
```

Comment:
> The design establishes Ethereum as the senior chain to preserve the existing security model while enabling point mobility.

Doesn't the added capability of point mobility radically alter the security model? That's a whole new surface area.

Reply:
I think the ideal case here is that we basically allow a star to push it's spawn proxy to the bridge and separate it's planets from it's point, such that in the earlier iterations of this we really are only having planets operate on other chains. This would require essentially adding bridge addresses to be addresses from which the ownership of a star could not revoke/reclaim the proxy unilaterally. This would require an onchain senate vote to modify.